File: App_medica/Citas_Medicas_API/src/app/API/models/citas.ts
/* tslint:disable */
/* eslint-disable */
import { Diagnosticos } from './diagnosticos';
import { Medicos } from './medicos';
import { Pacientes } from './pacientes';
export interface Citas {
  diagnostico?: Diagnosticos;
  fecha?: string;
  id?: number;
  medico?: Medicos;
  observaciones?: null | string;
  paciente?: Pacientes;
  usuariomedico?: null | string;
  usuariopaciente?: null | string;
}

File: Hospital/Models/Citas.cs
using System.ComponentModel.DataAnnotations;

namespace Hospital.Models
{
    public class Citas
    {
        [Key]
        public int Id { get; set; }
        public DateTime Fecha { get; set; }
        public string Observaciones { get; set; }

        public string usuariopaciente { get; set; }
        public virtual Pacientes Paciente { get; set; }

        public string usuariomedico { get; set; }
        public virtual Medicos Medico { get; set; }

        public virtual Diagnosticos Diagnostico { get; set; }
    }
}
